atlasdb-ete-tests/docker-compose.multiple-cassandra.yml

```diff
@@ -0,0 +1,31 @@
+version: '2'
+
+services:
+  cassandra1:
+     image: com.palantir.docker/atlas-cassandra:2.2-v0.1
+     environment:
+      - CASSANDRA_SEEDS=cassandra1,cassandra2
```

Comment:
We don't need 3 seeds as only one node is supposed to be down at a time, so when it comes up, it should still be able to become aware of all three by contacting to the seed node that was up.